deno_typescript/lib.rs

```diff
@@ -148,6 +148,7 @@ pub fn compile_bundle(
   let config_json = serde_json::json!({
     "compilerOptions": {
       "declaration": true,
+      "importsNotUsedAsValues": "error",
```

Comment:
Why set this to "error" and not "remove"? Maybe add explanation to comment.

Reply:
`"remove"` is the default.  It does not force us to actually make the code more "transpile" safe.  Where as `"error"` will alter us.  It breaks to many things to be the default for Deno runtime though.  I will add a comment.